daffodil-lib/src/main/scala/org/apache/daffodil/schema/annotation/props/ByHandMixins.scala

```diff
@@ -95,10 +95,34 @@ object TextNumberBase {
       case "8" => 8
       case "10" => 10
       case "16" => 16
-      case _ => self.schemaDefinitionError("Illegal number base: " + str) // validation will have checked. So this shoudn't happen.
+      case _ => self.schemaDefinitionError("For property textStandardBase, value must be 2, 8, 10, or 16. Found: %s", str)
```

Comment:
Aren't we also testing for this in ElementBaseGrammarMixin? Why do we need to check twice?

Reply:
Yep, the SDE in ElemenetBaseGrammarMixin should just be an assert.